qa/tasks/cephfs/fuse_mount.py

```diff
@@ -99,13 +99,20 @@ def _mount(self, mount_path, mount_fs_name, mount_options):
         run_cmd.extend(fuse_cmd)
 
         def list_connections():
+            conn_dir = "/sys/fs/fuse/connections"
+
+            conn_dir_doesnt_exists = bool(self.client_remote.run(
+                args=['stat', conn_dir], check_status=False).returncode)
```

Comment:
check_status=False, conn_dir_doesnt_exists  is always true?

Reply:
For RHEL and CentOS 8, I do this is always true since I see the message `/sys/fs/fuse/connections: mount point does not exist.` in logs everytime. However, that's not the case with Ubuntu, the tests completed running  successfully.

Here's the link to integration test run in case you want to take look -
http://pulpito.ceph.com/rishabh-2020-05-14_09:40:39-fs-wip-rishabh-15070-distro-basic-smithi/
http://pulpito.ceph.com/rishabh-2020-05-14_07:41:09-fs-wip-rishabh-15070-distro-basic-smithi/

Since tests run fine for Ubuntu I feel the issue might be with FUSE or ceph-fuse for RHEL.

PS: By tests I am referring to test_multifs.py testsuite that is present on PR #32581. I come up against this error while running that testsuite.